contracts/bondingcurve/BondingCurve.sol

```diff
@@ -32,6 +32,9 @@ contract BondingCurve is IBondingCurve, OracleRef, PCVSplitter, Timed, Incentivi
     /// @notice the discount applied on top of peg before at Scale
     uint256 public override discount;
 
+    /// @notice the cap on how much FEI can be minted by the bonding curve
+    uint256 public override mintCap;
```

Comment:
When we upgrade the contracts, are we doing it with a proxy/storage pattern or some other method?
If we are using a proxy/storage pattern, the variable should be added after BASIS_POINTS_GRANULARITY.

Reply:
We don't use proxy/storage its just drop-and-replace